examples/lujain_work_area/Contact_Book/contact_book.py

```diff
@@ -0,0 +1,27 @@
+# contact_book.py
+
+from contact_records import ContactRecords
+
+class ContactBook:
+    def __init__(self, csv_file):
+        self.contact_records = ContactRecords(csv_file)
+
+    def add_contact(self, name, phone, email, address):
+        contact = {'Name': name, 'Phone': phone, 'Email': email, 'Address': address}
```

Comment:
This method should add a ContactRecord rather than a dictionary. The reason for this is, we can make changes only to ContactRecord class if we get any future requirements from client, avoiding changing the logic for in existing code.